qadence/backends/pyqtorch/config.py

```diff
@@ -4,7 +4,7 @@
 from logging import getLogger
 from typing import Callable
 
-from pyqtorch.utils import SolverType
+from pyqtorch.utils import DropoutMode, SolverType
```

Comment:
You need to expose the `DropoutMode` in Qadence by adding it to the `types` as well.